src/components/atoms/Icon/Icons/sheet-file.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="22" height="20" viewBox="0 0 22 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M1.34343 5.8147H5.66213C6.40272 5.8147 7.00509 5.21902 7.00509 4.48656V1.32837C7.00509 0.595913 6.40248 0 5.66165 0H1.34296C0.60237 0 0 0.595674 0 1.32789V4.48633C0 5.21878 0.602609 5.8147 1.34343 5.8147ZM5.09209 3.9017H1.91309V1.91333H5.09209V3.9017Z" fill="#C7C5C5"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M10.006 5.8147H20.698C21.4158 5.8147 22 5.23313 22 4.51837V1.29633C22 0.581565 21.4158 0 20.6977 0H10.0055C9.28763 0 8.70343 0.581565 8.70343 1.29633V4.51813C8.70343 5.23313 9.28763 5.8147 10.006 5.8147ZM20.0869 3.9017H10.6164V1.91333H20.0869V3.9017Z" fill="#C7C5C5"/>
```

Comment:
I mentioned before, but we need all svg's fill values to be `currentColor` so that we can easily change the color from within the Icon component.